mtags/src/main/scala-2/scala/meta/internal/pc/completions/AmmoniteCompletions.scala

```diff
@@ -106,6 +108,72 @@ trait AmmoniteCompletions { this: MetalsGlobal =>
       }
   }
 
+  case class AmmoniteIvyCompletions(
+      select: Tree,
+      selector: List[ImportSelector],
+      pos: Position,
+      editRange: l.Range
+  ) extends CompletionPosition {
+
+    override def contribute: List[Member] =
+      try {
+
+        val query = selector.collectFirst {
+          case sel: ImportSelector if sel.name.toString().contains(CURSOR) =>
+            sel.name.decode.replace(CURSOR, "")
+        }
+
+        query match {
+          case Some(imp) =>
+            val api = coursierapi.Complete
+              .create()
+              .withScalaVersion("2.13.8")
+              .withScalaBinaryVersion("2.13")
```

Comment:
```suggestion
```
is this needed? Can't coursier infer it itself?

We could also use `ScalaVersions.scalaBinaryVersionFromFullVersion` or just inline that method.

Reply:
Yeah that makes sense, but I think it's necessary atm, see: https://github.com/coursier/interface/pull/274 so I've inlined it.